src/Concerns/WaitsForElements.php

```diff
@@ -86,6 +86,22 @@ public function waitForText($text, $seconds = null)
         }, $message);
     }
 
+    /**
+     * @param string $selector
```

Comment:
Please format the PHPDoc properly. Take a look at `waitForText()`.

Reply:
Changed it in my Fork, should I start a new PR?